string name #name of joint
uint16[] port_values #array of port GPIO values [0,1] for digital or [0,4095] for analog